compose/ui/ui/build.gradle

```diff
@@ -318,6 +319,11 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 kotlin.srcDir("src/webTest/kotlin")
                 dependsOn(skikoTest)
             }
+            uikitInstrumentedTest.dependencies {
+                implementation(project(":compose:material:material"))
+                implementation(project(":compose:foundation:foundation"))
+                implementation(project(":compose:ui:ui-test-junit4"))
```

Comment:
"junit4" isn't supposed to be used outside of JVM

Reply:
Removed as unnecessary import, however it runs on iOS, including skiko tests...